#Configuration file for laser shade occlusion manager
{
    occlusion_manager_number_lasers: 1,
    use_polygon_occlusion_manager: true,
    occlusion_manager_visualization_enabled: false,
    occlusion_manager_time_uncertainty_factor: 1.3,
    occlusion_manager_max_range: 20,
    occlusion_self_occlusion_distance: 0.25,
    max_misses_before_deletion: 10,
    max_misses_before_deletion_of_mature_track: 30,
    occlusion_manager_subscriber_queue_size: 100,
    occlusion_manager_sync_buffer_size: 100,
    occlusion_allowed_duration_for_reappearance: 5.0,
    occlusion_manager_maximum_sync_slop: 0.1,
    occlusion_manager_minimum_absolute_velocity: 0.1,
    occlusion_manager_number_of_reappeared_frames_to_zero_velocity: 5,
    occlusion_geodesics_motion_variance: 1.0,
    occlusion_geodesics_total_cut_off: 0.4,
    occlusion_geodesics_plausibility_cut_off: 0.01,
    occlusion_geodesics_inertia_variance: 0.1,
    occlusion_geodesics_use_detection_probability: 0,
    occlusion_geodesics_use_plausibility: 1,
    occlusion_geodesics_use_inverted_plausibility: 1,
    occlusion_geodesics_use_duration_cost: 1,
    occlusion_geodesics_scale_reassignment: 0,
}
